dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaOutput.java

```diff
@@ -33,18 +33,14 @@ public KafkaOutput(String topic, KafkaProducer<K, V> kafkaProducer) {
     }
 
     public SubscriberBuilder<Message<V>, Void> getSubscriber() {
-        return ReactiveStreams.<Message<V>> builder().takeWhile(m -> running).onError(this::logError).forEach(this::sendMessage);
+        return ReactiveStreams.<Message<V>> builder().takeWhile(m -> running).onError(this::reportSendException).forEach(this::sendMessage);
```

Comment:
`reportSendException` isn't quite right either - the error didn't occur while sending a message to kafka, it occurred at some point earlier in the stream.

I think we need a new message for this.

Reply:
Changing to this
kafka.output.error.signal.CWMRX1004E=CWMRX1004E: The Reactive Messaging Kafka Outgoing Connector received an error. The error is {0}.